mods/qtcore/models/Center_Cube.obj

```diff
@@ -0,0 +1,39 @@
+# Blender 4.1.1
```

Comment:
What is this?

Reply:
Ah, this is a 3D model file. .obj files can be several things, including:
- 3D model (you can tell because this format is actually text)
- compiled program code (this is definitely not text)